[Defaults]: Awards (Golden Globes filters)

### Link to File

award (golden globes)

### Version Number

2.2.0-build37

### What branch are you on?

nightly

### Describe the Bug

Same issue with Best Director Winners for Golden Globes.

[2025-08-04 09:07:35,841] [builder.py:205]            [INFO]     |================================================================================================================================================================================================|
[2025-08-04 09:07:35,841] [builder.py:205]            [INFO]     |                                                                    Golden Globes Best Picture Winners Collection in Movies                                                                     |
[2025-08-04 09:07:35,841] [builder.py:205]            [INFO]     |================================================================================================================================================================================================|
[2025-08-04 09:07:35,841] [builder.py:206]            [INFO]     |                                                                                                                                                                                                |
[2025-08-04 09:07:35,841] [builder.py:217]            [INFO]     |============================================================================== Building Definition From Templates ==============================================================================|
[2025-08-04 09:07:35,841] [builder.py:218]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,841] [builder.py:228]            [DEBUG]    | Templates Called: shared, arr, custom                                                                                                                                                          |
[2025-08-04 09:07:35,841] [builder.py:229]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,841] [builder.py:232]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,842] [builder.py:233]            [DEBUG]    | Validating Method: variables                                                                                                                                                                   |
[2025-08-04 09:07:35,843] [meta.py:367]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,843] [meta.py:368]               [DEBUG]    | Conditional: name_<<key>>                                                                                                                                                                      |
[2025-08-04 09:07:35,844] [meta.py:367]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,844] [meta.py:368]               [DEBUG]    | Conditional: summary_<<key>>                                                                                                                                                                   |
[2025-08-04 09:07:35,844] [meta.py:375]               [DEBUG]    | Conditional Variable: summary_best_picture overwritten to "The Golden Globe Award for Best Motion Picture as awarded by the Hollywood Foreign Press Association.  Contains winners of Animation, Drama, and Musical or Comedy." |
[2025-08-04 09:07:35,844] [meta.py:367]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,844] [meta.py:368]               [DEBUG]    | Conditional: url_poster_<<key>>                                                                                                                                                                |
[2025-08-04 09:07:35,844] [meta.py:367]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,844] [meta.py:368]               [DEBUG]    | Conditional: final_use                                                                                                                                                                         |
[2025-08-04 09:07:35,888] [meta.py:598]               [DEBUG]    |================================================================================== Final Template Attributes ===================================================================================|
[2025-08-04 09:07:35,888] [meta.py:599]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,888] [meta.py:600]               [DEBUG]    | {'run_definition': ['movie'], 'translation_key': 'golden_picture', 'summary': 'The Golden Globe Award for Best Motion Picture as awarded by the Hollywood Foreign Press Association.  Contains winners of Animation, Drama, and Musical or Comedy.', 'ignore_blank_results': True, 'item_radarr_tag': 'pmm_golden_best_picture', 'sort_title': '!020_Golden Globe !1', 'url_poster': 'https://raw.githubusercontent.com/Kometa-Team/Default-Images/master/award/golden/best_picture_winner.jpg', 'file_background': 'config/assets/defaults/background.jpg', 'radarr_add_missing': True, 'radarr_monitor': False, 'collection_order': 'release.desc', 'sync_mode': 'sync', 'cache_builders': 0} |
[2025-08-04 09:07:35,888] [meta.py:601]               [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,888] [builder.py:245]            [INFO]     |=================================================================== Validating Golden Globes Best Picture Winners Attributes ===================================================================|
[2025-08-04 09:07:35,888] [builder.py:310]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:35,889] [builder.py:311]            [DEBUG]    | Validating Method: translation_key                                                                                                                                                             |
[2025-08-04 09:07:35,889] [builder.py:314]            [DEBUG]    | Value: golden_picture                                                                                                                                                                          |
[2025-08-04 09:07:35,889] [builder.py:394]            [INFO]     | Final Name: Golden Globes Best Picture Winners                                                                                                                                                 |
[2025-08-04 09:07:37,479] [builder.py:437]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,479] [builder.py:438]            [DEBUG]    | Validating Method: run_definition                                                                                                                                                              |
[2025-08-04 09:07:37,479] [builder.py:441]            [DEBUG]    | Value: ['movie']                                                                                                                                                                               |
[2025-08-04 09:07:37,480] [builder.py:641]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,480] [builder.py:642]            [DEBUG]    | Validating Method: ignore_blank_results                                                                                                                                                        |
[2025-08-04 09:07:37,480] [builder.py:643]            [DEBUG]    | Value: True                                                                                                                                                                                    |
[2025-08-04 09:07:37,480] [builder.py:713]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,480] [builder.py:714]            [DEBUG]    | Validating Method: delete_collections_named                                                                                                                                                    |
[2025-08-04 09:07:37,480] [builder.py:715]            [DEBUG]    | Value: ['Golden Globe Best Motion Pictures']                                                                                                                                                   |
[2025-08-04 09:07:37,486] [plex.py:1081]              [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,486] [plex.py:1084]              [DEBUG]    | Looking for: Golden Globe Best Motion Pictures                                                                                                                                                 |
[2025-08-04 09:07:37,486] [builder.py:757]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,486] [builder.py:758]            [DEBUG]    | Validating Method: sync_mode                                                                                                                                                                   |
[2025-08-04 09:07:37,487] [builder.py:762]            [DEBUG]    | Value: sync                                                                                                                                                                                    |
[2025-08-04 09:07:37,487] [builder.py:967]            [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,487] [builder.py:968]            [DEBUG]    | Validating Method: collection_order                                                                                                                                                            |
[2025-08-04 09:07:37,487] [builder.py:969]            [DEBUG]    | Value: release.desc                                                                                                                                                                            |
[2025-08-04 09:07:37,487] [builder.py:1000]           [DEBUG]    |                                                                                                                                                                                                |
[2025-08-04 09:07:37,487] [builder.py:1004]           [DEBUG]    | Validating Method: imdb_award                                                                                                                                                                  |
[2025-08-04 09:07:37,487] [builder.py:1005]           [DEBUG]    | Value: {'event_id': 'ev0000292', 'event_year': 'all', 'category_filter': ['best motion picture - animated', 'best motion picture - comedy', 'best motion picture - comedy or musical', 'best motion picture - drama', 'best motion picture - foreign language', 'best motion picture - musical', 'best motion picture - musical or comedy', 'best motion picture - non-english language', 'best motion picture, animated', 'best motion picture, drama', 'best motion picture, musical or comedy', 'best motion picture, non-english language'], 'winning': True} |
[2025-08-04 09:07:40,625] [kometa.py:971]             [DEBUG]    | Traceback (most recent call last):                                                                                                                                                             |
                                                                 |   File "//kometa.py", line 829, in run_collection                                                                                                                                              |
                                                                 |     builder = CollectionBuilder(config, metadata, mapping_name, collection_attrs, library=library, extra=output_str)                                                                           |
                                                                 |   File "/modules/builder.py", line 1076, in __init__                                                                                                                                           |
                                                                 |     self._imdb(method_name, method_data)                                                                                                                                                       |
                                                                 |     ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                       |
                                                                 |   File "/modules/builder.py", line 1600, in _imdb                                                                                                                                              |
                                                                 |     raise Failed(f"{self.Type} Error: imdb_award category_filter attribute invalid: {category_filter} must be in in [{', '.join([v for _, v in category_names.items()])}]")                    |
                                                                 |                                                                                                                                                ^^^^^^^^^^^^^^^^^^^^                            |
                                                                 | AttributeError: 'list' object has no attribute 'items'                                                                                                                                         |
                                                                 |                                                                                                                                                                                                |
[2025-08-04 09:07:40,626] [kometa.py:972]             [ERROR]    | Unknown Error: 'list' object has no attribute 'items'                                                                                                                                          |
[2025-08-04 09:07:40,626] [kometa.py:977]             [INFO]     |                                                                                                                                                                                                |
[2025-08-04 09:07:40,626] [kometa.py:978]             [INFO]     |================================================================================================================================================================================================|
[2025-08-04 09:07:40,626] [kometa.py:978]             [INFO]     |                                                                     Finished Golden Globes Best Picture Winners Collection                                                                     |
[2025-08-04 09:07:40,626] [kometa.py:978]             [INFO]     |                                                                                  Collection Run Time: 0:00:04                                                                                  |
[2025-08-04 09:07:40,626] [kometa.py:978]             [INFO]     |================================================================================================================================================================================================|

--- a/defaults/award/golden.yml
+++ b/defaults/award/golden.yml
@@ -36,10 +36,7 @@ collections:
         - best motion picture - musical
         - best motion picture - musical or comedy
         - best motion picture - non-english language
-        - best motion picture, animated
-        - best motion picture, drama
         - best motion picture, musical or comedy
-        - best motion picture, non-english language
       winning: true
 
   Golden Globes Best Director Winners:
@@ -59,7 +56,6 @@ collections:
       category_filter:
         - best director
         - best director - motion picture
-        - best director, motion picture
       winning: true
     
 dynamic_collections:

